src/main/java/com/icthh/xm/ms/entity/service/json/JsonListenerConfiguration.java

```diff
@@ -0,0 +1,54 @@
+package com.icthh.xm.ms.entity.service.json;
+
+import com.icthh.xm.commons.config.client.api.RefreshableConfiguration;
+import com.icthh.xm.ms.entity.service.spec.XmEntitySpecContextService;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.util.AntPathMatcher;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class JsonListenerConfiguration implements RefreshableConfiguration {
+
+    private final String mappingPath;
+    private final AntPathMatcher matcher;
+    private final XmEntitySpecContextService xmEntitySpecContextService;
+    private final JsonListenerService jsonListenerService;
+
+    private static final String TENANT_NAME = "tenantName";
+
+    public JsonListenerConfiguration(@Value("${spring.application.name}") String appName,
```

Comment:
May be  rename JsonListenerConfiguration -> JsonConfigurationListener

Reply:
Agree. Sounds better. Thanks